administration/src/cards/pdf/PdfQrCodeElement.ts

```diff
@@ -27,6 +28,11 @@ const pdfQrCodeElement: PdfElement<PdfQrCodeElementProps, PdfQrCodeElementRender
     qrCode: qrCode,
   }).toBinary()
 
+  // Log base64 activationCode for development purposes
+  if (qrCode.case === 'dynamicActivationCode' && window.location.hostname === 'localhost') {
```

Comment:
Maybe you can create a helper constant somewhere to reuse this. Sth like:
`const DEV_MODE = window.location.hostname === 'localhost'`